src/crewai/cli/enterprise/main.py

```diff
@@ -0,0 +1,84 @@
+import requests
+from typing import Dict, Any
+from rich.console import Console
+from requests.exceptions import RequestException, JSONDecodeError
+
+from crewai.cli.command import BaseCommand
+from crewai.cli.settings.main import SettingsCommand
+from crewai.cli.version import get_crewai_version
+
+console = Console()
+
+
+class EnterpriseCommand(BaseCommand):
```

Comment:
`EnterpriseConfigureCommand` would be better? Since enterprise is the name of the group of commands. 

Really small detail, don't need to change if you are not convinced haha